app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
+        try:
+            cafe.visit_cafe(friends[i])
+        except (NotVaccinatedError, OutdatedVaccineError):
+            return 'All friends should be vaccinated'
+        except NotWearingMaskError:
+            total += 1
+
+    if total:
+        return f'Friends should buy {total} masks'
```

Comment:
The variable total is a bit misleading. It's actually counting the number of friends not wearing masks, so a more descriptive name like unmasked_count might be clearer.